site/docs/master/configuration.md

```diff
@@ -1,9 +1,54 @@
 # Contour Configuration
 
+## Configuration File
+
```

Comment:
Should we also explain the hierarchy of what values override which? Meaning if I pass say `incluster` in the config file but also set the arg to `contour serve`, what happens?

Reply:
Good point, I'll add that.